app/utils/store.py

```diff
@@ -101,23 +105,24 @@ def generate_encryption_key(self):
     def get_document_key(self, service_id, document_id):
         return "{}/{}".format(service_id, document_id)
 
-    def _hash_recipient_email(self, verification_email):
-        """
-        We pass in verification_email, which is recipient's email address.
```

Comment:
We don't want the docstring anymore?

Reply:
Oop, we do! I will add it back in hasher.py which feels like the place to have it :) Good spot 👀